### DISCUSSION TITLE
Problem with import job queue

### DISCUSSION THREAD

**Discussion by NaungYeHtet** [filamentphp/filament#12088] [Help] [answered]
### Package

Actions

### Package Version

v3.2

### How can we help you?

When the queue connection is database, there is no job being executed when imported.

<img width="398" alt="image" src="https://github.com/filamentphp/filament/assets/49832780/b2dbb85e-cb22-4af3-9d37-9659c72d9ad6">
<img width="535" alt="image" src="https://github.com/filamentphp/filament/assets/49832780/db7e97e8-db51-4481-8c79-c7703560b1bc">
<img width="559" alt="image" src="https://github.com/filamentphp/filament/assets/49832780/ed015c7f-5ddd-4b3d-9fe1-7c2d2da9c6a6">

When the queue connection is redis, there are several Job keeps retrying... 

<img width="232" alt="image" src="https://github.com/filamentphp/filament/assets/49832780/5cb4978c-cd57-4b8e-8a67-6e6189510251">
<img width="826" alt="image" src="https://github.com/filamentphp/filament/assets/49832780/42f2e330-8579-41e8-ac13-0edee3337ab1">
<img width="526" alt="image" src="https://github.com/filamentphp/filament/assets/49832780/08bad8d2-84ba-495d-b2af-4db29c672ad4">

**Comment by sasori13L** [comment, marked as answer]
What Laravel version are you using? I have a feeling you are using Laravel 11, I too had the same issue which was weird cause the import action worked on Laravel 10 without issues. Then I had to check the Laravel 10 QUEUE_CONNECTION it was set to "sync" while Laravel 11 has "database" as the value. When I changed it to "sync" everything works now.

**Reply by VTrace** [reply]
It works for me, thanks..
